.github/scripts/ffmpeg/execute-tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+SOURCE_PATH=$(realpath `dirname ${BASH_SOURCE[0]}`/../../../code)
+# path to the file with test names which should be skipped
+SKIP_TESTS_PATH=$(realpath `dirname ${BASH_SOURCE[0]}`/skip-tests.txt)
+
+echo "::group::Test FFmpeg"
+    pushd $FFMPEG_TESTS_PATH
+
+    mkdir -p tests/data/fate
+    ln -s /usr/bin/base64 tests/base64
```

Comment:
This line fails when the script is executed multiple time (e.g. locally). I'd suggest to move it to \`unpack-tests.sh\`

Reply:
unpack-tests is powershell script. The issue has been fixed with a check.